spec/models/google_analytics/data_tracking_spec.rb

```diff
@@ -13,10 +13,23 @@ module GoogleAnalytics
           allow(described_class).to receive(:adapter).and_return('Adapter')
         end
 
-        %w(staging production).each do |host|
-          it "returns true when host is #{host}" do
-            allow(RailsHost).to receive(:env).and_return(host)
-            expect(described_class.enabled?).to be_truthy
+        context 'with usage tracking accepted' do
```

Comment:
see https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3839/files#r603201741